definitions/pipelines/somatic_wgs.cwl

```diff
@@ -0,0 +1,529 @@
+    vep_summary:
+        type: File
+        outputSource: detect_variants/vep_summary
+    tumor_snv_bam_readcount_tsv:
+        type: File
+        outputSource: detect_variants/tumor_snv_bam_readcount_tsv
+    tumor_indel_bam_readcount_tsv:
+        type: File
+        outputSource: detect_variants/tumor_indel_bam_readcount_tsv
+    normal_snv_bam_readcount_tsv:
+        type: File
+        outputSource: detect_variants/normal_snv_bam_readcount_tsv
+    normal_indel_bam_readcount_tsv:
+        type: File
+        outputSource: detect_variants/normal_indel_bam_readcount_tsv
+# ##copy number calling
+#     intervals_antitarget:
+#         type: File?
+#         outputSource: cnvkit/intervals_antitarget
+#     intervals_target:
+#         type: File?
+#         outputSource: cnvkit/intervals_target
+#     normal_antitarget_coverage:
+#         type: File
+#         outputSource: cnvkit/normal_antitarget_coverage
+#     normal_target_coverage:
+#         type: File
+#         outputSource: cnvkit/normal_target_coverage
+#     reference_coverage:
+#         type: File?
+#         outputSource: cnvkit/reference_coverage
+#     cn_diagram:
+#         type: File?
+#         outputSource: cnvkit/cn_diagram
+#     cn_scatter_plot:
+#         type: File?
+#         outputSource: cnvkit/cn_scatter_plot
+#     tumor_antitarget_coverage:
+#         type: File
+#         outputSource: cnvkit/tumor_antitarget_coverage
+#     tumor_target_coverage:
+#         type: File
+#         outputSource: cnvkit/tumor_target_coverage
+#     tumor_bin_level_ratios:
+#         type: File
+#         outputSource: cnvkit/tumor_bin_level_ratios
+#     tumor_segmented_ratios:
+#         type: File
+#         outputSource: cnvkit/tumor_segmented_ratios
+##sv calling
+    diploid_variants:
+        type: File?
+        outputSource: manta/diploid_variants
+        secondaryFiles: [.tbi]
+    somatic_variants:
+        type: File?
+        outputSource: manta/somatic_variants
+        secondaryFiles: [.tbi]
+    all_candidates:
+        type: File
+        outputSource: manta/all_candidates
+        secondaryFiles: [.tbi]
+    small_candidates:
+        type: File
+        outputSource: manta/small_candidates
+        secondaryFiles: [.tbi]
+    tumor_only_variants:
+        type: File?
+        outputSource: manta/tumor_only_variants
+        secondaryFiles: [.tbi]
+##sample concordance check
+    somalier_concordance_metrics:
+        type: File
+        outputSource: concordance/somalier_pairs
+    somalier_concordance_statistics:
+        type: File
+        outputSource: concordance/somalier_samples
+steps:
+    tumor_alignment_and_qc:
+        run: wgs_alignment.cwl
+        in:
+            reference: reference
+            bams: tumor_bams
+            readgroups: tumor_readgroups
+            mills: mills
+            known_indels: known_indels
+            dbsnp_vcf: dbsnp_vcf
+            omni_vcf: omni_vcf
+            intervals: qc_intervals ##todo
```

Comment:
What is this `##todo` marking?

Reply:
My mistake, I needed to remove it. The somatic exome pipeline used the qc_interval list for exome alone, i needed to change the file with the respective whole genome.